hyperspy/tests/signal/test_2D_tools.py

```diff
@@ -122,3 +124,15 @@ def test_align_expand(self):
         # Check alignment is correct
         d_al = s.data[:, ds[0]:-ds[0], ds[1]:-ds[1]]
         nt.assert_true(np.all(d_al == self.aligned))
+
+
+class TestPhaseFunctions:
+
+    def test_add_phase_ramp(self):
+        self.s = hs.signals.Signal2D(np.exp(1j * (np.indices((3, 3)).sum(axis=0) + 4)))
+        self.s.add_phase_ramp(-1, -1, -4)
+        npt.assert_almost_equal(self.s.angle().data, 0)
+
+
+if __name__ == '__main__':
```

Comment:
From a terminal/console, this would be easy, but I normally use the PyCharm IDE where executing a script is just one click, so it is at least easier for me. Should I delete them?